### ISSUE TITLE
[chrome/jellybean 4.1.2] Sporadic JS error breaks /sign_in when bringing up the dialog with authed users.

### ISSUE THREAD

**Issue by jrgm** [mozilla/persona#2627]
I was testing with Chrome (18.0.1025308) on JellyBean (4.1.2) and noticed some flakiness in bringing up the dialog.

STR
1. I had two authed users on an account: one @gmail.com and the other @eyedee.me. The @eyedee.me one was the one last used with beta.123done.org, so preselected when the dialog comes up.
2. What would happen was that the dialog would come up blank. Not everytime but maybe one in four.
3. I think I had a good network connection. No flakiness there.

I tried this repetively and when I got the blank dialog, `adb logcat` would report:

```
I/chromium( 1379): [INFO:CONSOLE(3)] "Uncaught TypeError: Cannot read property 'd' of undefined", source: https://static.login.anosrep.org/v/02c3d2117d/production/en/dialog.js (3)
```

Which, with minified code in stage, is a pretty useless error message in actually figuring out what's wrong. I should spend some time trying to do this with dev to get a better message. But just filing this bug for now; I may get to dev sometime later, but if anyone has copious free time, please see if you can get the same error.

**Comment by jrgm** [comment]
Link to GH-2629

**Comment by jrgm** [comment]
I tried this on two different android phones running chrome and couldn't reproduce this on either:

Galaxy SII - Android 4.0.4, Chrome 18.0.1025123
HTC Evo   - Android 4.0.4, Chrome 18.0.1025308

**Comment by lloyd** [comment]
please verify fixed in dev and thus is a dup of issue #2629

**Comment by jrgm** [comment]
Hmm, verified but not closed. Closing now.
